src/plugins/persistence/couch/CouchObjectProvider.js

```diff
@@ -219,7 +224,12 @@ class CouchObjectProvider {
                 console.error(error.message);
                 throw new Error(`CouchDB Error - No response"`);
             } else {
-                console.error(error.message);
+                if (!isNotebookOrAnnotationType(body.model)) {
```

Comment:
Let's make this a positive condition for readability.

Reply:
Also this is potentially undefined as not all requests will have a body so we should check for that